webview-ui/src/components/ProfileManager/ProfileEditorForm.tsx

```diff
@@ -0,0 +1,360 @@
+                {nameErrorMsg}
+              </HelperTextItem>
+            </HelperText>
+          </FormHelperText>
+        )}
+      </FormGroup>
+
+      <FormGroup label="Target Technologies" fieldId="targets">
+        <Select
+          isOpen={targetOpen}
+          onOpenChange={setTargetOpen}
+          onSelect={(_ev, value) => handleTargetsChange(value as string)}
+          toggle={(ref) => (
+            <MenuToggle
+              ref={ref}
+              style={{
+                minWidth: "250px",
+                maxWidth: "400px",
+                overflow: "hidden",
+                textOverflow: "ellipsis",
+                whiteSpace: "nowrap",
+              }}
+              isDisabled={profile.readOnly}
+              onClick={handleTargetToggle}
+              isExpanded={targetOpen}
+            >
+              {selectedTargets.length ? selectedTargets.join(", ") : "Select targets"}
+            </MenuToggle>
+          )}
+          selected={selectedTargets}
+        >
+          <SelectList style={{ maxHeight: "200px", overflowY: "auto" }}>
+            {targetOptions.map((opt) => (
+              <SelectOption key={opt} value={opt}>
+                {opt}
+              </SelectOption>
+            ))}
+          </SelectList>
+        </Select>
+      </FormGroup>
+
+      <FormGroup label="Source Technologies" fieldId="sources">
+        <Select
+          isOpen={sourceOpen}
+          onOpenChange={setSourceOpen}
+          onSelect={(_ev, value) => handleSourcesChange(value as string)}
+          toggle={(ref) => (
+            <MenuToggle
+              ref={ref}
+              style={{
+                minWidth: "250px",
+                maxWidth: "400px",
+                overflow: "hidden",
+                textOverflow: "ellipsis",
+                whiteSpace: "nowrap",
+              }}
+              isDisabled={profile.readOnly}
+              onClick={handleSourceToggle}
+              isExpanded={sourceOpen}
+            >
+              {selectedSources.length ? selectedSources.join(", ") : "Select sources"}
+            </MenuToggle>
+          )}
+          selected={selectedSources}
+        >
+          <SelectList style={{ maxHeight: "200px", overflowY: "auto" }}>
+            {sourceOptions.map((opt) => (
+              <SelectOption key={opt} value={opt}>
+                {opt}
+              </SelectOption>
+            ))}
+          </SelectList>
+        </Select>
+      </FormGroup>
+
+      <FormGroup label="Use Default Rules" fieldId="use-default-rules">
+        <Switch
+          id="use-default-rules"
+          isChecked={localProfile.useDefaultRules}
+          isDisabled={profile.readOnly}
+          onChange={(_e, checked) => {
+            const updated = { ...localProfile, useDefaultRules: checked };
+            setLocalProfile(updated);
+            debouncedChange(updated);
+          }}
+        />
+      </FormGroup>
+      {/* //TODO */}
```

Comment:
TODO: Determine how/if we want to be able to configure analysis mode here.

Reply:
I dont think analysis mode should be exposed here. I do not see we ever doing a full analysis for Kai, we are only doing source only